cmd/zetae2e/local/local.go

```diff
@@ -322,6 +322,8 @@ func localE2ETest(cmd *cobra.Command, _ []string) {
 				e2etests.TestPrecompilesPrototypeThroughContractName,
 				e2etests.TestPrecompilesStakingName,
 				e2etests.TestPrecompilesStakingThroughContractName,
+				e2etests.TestPrecompilesBankName,
+				e2etests.TestPrecompilesBankFailName,
```

Comment:
Can we maybe have a test doing it through smart contract?

Reply:
Working on it ATM.